doc/developer-guide/api/functions/TSHttpEffectiveUrlStringGet.en.rst

```diff
@@ -0,0 +1,42 @@
+.. Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I think this could go into e.g. doc/developer-guide/api/functions/TSUrlStringGet.en.rst, I much rather prefer grouping similar APIs into a single file, rather than one file per API.